klt-cert-manager/config/rbac/role_binding.yaml

```diff
@@ -16,4 +16,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: cert-manager
-  namespace: system
+  namespace: system
```

Comment:
```suggestion
  namespace: system

```

Reply:
this is autogenerated :( kubebuilder removes it all the time